NOTES.md

```diff
@@ -0,0 +1,36 @@
+# Things Paul Came Across
+
+## General Python housekeeping I could do later
+
+- *Proper package*. Make imports more normal by changing `t-html` to something legally importable.
+- *pyproject.toml*. Put dependencies in there, then put everything under `src/thtml`. This would mean moving the
```

Comment:
this works if I don't plan to add `tsql` and/or `tcss` and/or ... `t-something` ... any idea how we could actually have each folder represent just a single module/package?

Reply:
It depends on whether you want to ship the packages separately. If so, it's a mono-repo style. I could propose something for that.

Reply:
the initial idea (we talked about) was to use a generic `t-750` name to have more than "*just*" `thml` in it but I wouldn't mind going one step at the time and create a polished (after your changes are in) dedicated repo for *just that* ... if we keep things separate then *git submodules* work very well to recreate, eventually, that *monorepo* so let's agree on the name and migrate into that repo instead, and keep this for future updates/possibilites?

Reply:
I agree, and maybe the repos are connected together in a GitHub organization for all the t*** packages. For now: we'll work towards a package named something like `thtml` (but of course, [that is taken on PyPI](https://pypi.org/project/thtml/) .)